src/ngraph/runtime/gpu/gpu_external_function.cpp

```diff
@@ -241,7 +242,18 @@ runtime::gpu::GPU_ExternalFunction::GPU_ExternalFunction(
     : ngraph::runtime::ExternalFunction(function, release_function)
     , m_compiled_function(nullptr)
     , m_emit_timing(std::getenv("NGRAPH_GPU_EMIT_TIMING") != nullptr)
+    , m_ctx(new GPURuntimeContext)
 {
+    // Create context use driver API and make it current, the runtime call will pickup the context
+    // http://docs.nvidia.com/cuda/cuda-c-programming-guide/index.html
+    // #interoperability-between-runtime-and-driver-apis
+    ngraph::runtime::gpu::CudaContextManager::instance();
```

Comment:
The cudaInit function needs to be called before the gpu_emitter functions are called, since some of the cuda kernels are now compiled there. Unfortunately, the call_frame does not exist until after the gpu_emitters are run, and so this setup call has to happen earlier. That said because of @rkimballn1's backend change I suspect we will refactor this at some point.

Reply:
OK, let do it later.